string name
float64 price
int64 stock
---
string feedback